src/Common/src/System/IO/Win32Marshal.cs

```diff
@@ -1,136 +0,0 @@
-                        return new UnauthorizedAccessException(SR.Format(SR.UnauthorizedAccess_IODenied_Path, path));
-
-                case Interop.Errors.ERROR_ALREADY_EXISTS:
-                    if (path.Length == 0)
-                        goto default;
-
-                    return new IOException(SR.Format(SR.IO_AlreadyExists_Name, path), MakeHRFromErrorCode(errorCode));
-
-                case Interop.Errors.ERROR_FILENAME_EXCED_RANGE:
-                    return !string.IsNullOrEmpty(path) ?
-                        new PathTooLongException(SR.Format(SR.IO_PathTooLong_Path, path)) :
-                        new PathTooLongException(SR.IO_PathTooLong);
-
-                case Interop.Errors.ERROR_INVALID_PARAMETER:
-                    return new IOException(GetMessage(errorCode), MakeHRFromErrorCode(errorCode));
-
-                case Interop.Errors.ERROR_SHARING_VIOLATION:
-                    if (path.Length == 0)
-                        return new IOException(SR.IO_SharingViolation_NoFileName, MakeHRFromErrorCode(errorCode));
-                    else
-                        return new IOException(SR.Format(SR.IO_SharingViolation_File, path), MakeHRFromErrorCode(errorCode));
-
-                case Interop.Errors.ERROR_FILE_EXISTS:
-                    if (path.Length == 0)
-                        goto default;
-
-                    return new IOException(SR.Format(SR.IO_FileExists_Name, path), MakeHRFromErrorCode(errorCode));
-
-                case Interop.Errors.ERROR_OPERATION_ABORTED:
-                    return new OperationCanceledException();
-
-                default:
-                    return new IOException(GetMessage(errorCode), MakeHRFromErrorCode(errorCode));
-            }
-        }
-
-        /// <summary>
-        /// If not already an HRESULT, returns an HRESULT for the specified Win32 error code.
-        /// </summary>
-        internal static int MakeHRFromErrorCode(int errorCode)
-        {
-            // Don't convert it if it is already an HRESULT
-            if ((0xFFFF0000 & errorCode) != 0)
-                return errorCode;
-
-            return unchecked(((int)0x80070000) | errorCode);
-        }
-
-        /// <summary>
-        ///     Returns a Win32 error code for the specified HRESULT if it came from FACILITY_WIN32
-        ///     If not, returns the HRESULT unchanged
-        /// </summary>
-        internal static int TryMakeWin32ErrorCodeFromHR(int hr)
-        {
-            if ((0xFFFF0000 & hr) == 0x80070000)
-            {
-                // Win32 error, Win32Marshal.GetExceptionForWin32Error expects the Win32 format
-                hr &= 0x0000FFFF;
-            }
-
-            return hr;
-        }
-
-        /// <summary>
-        ///     Returns a string message for the specified Win32 error code.
-        /// </summary>
-        internal static string GetMessage(int errorCode)
-        {
-            return Interop.Kernel32.GetMessage(errorCode);
-        }
-    }
-}
```

Comment:
I assume you diff'd the files to validate there's nothing of value in this copy that should be transferred?